keyvi/include/keyvi/dictionary/fsa/internal/json_value_store.h

```diff
@@ -273,12 +269,10 @@ class JsonValueStoreMerge final : public JsonValueStoreMinimizationBase {
   }
 
   void Write(std::ostream& stream) {
-    boost::property_tree::ptree pt;
-    pt.put("size", std::to_string(values_buffer_size_));
-    pt.put("values", std::to_string(number_of_values_));
-    pt.put("unique_values", std::to_string(number_of_unique_values_));
+    // TODO(hendrik) write compressor
```

Comment:
Just checking: This is not required before merging?

Reply:
no, I noticed this being broken, but it is broken already. We basically loose information about applied compression when merging 2 files. But that information has no functional impact, it's only meta-information to know whether compression has been enabled when compiling or not.